### PR TITLE
Create global css

### LINKED ISSUE #3: Create global css
Add variables for color and font

### PR DESCRIPTION
**Pull request by Seborider**
closes #3
- Add fonts
- Define global variables

### CODE REVIEW

**Review by phlgr — APPROVED** [review]

File: src/app/globals.css [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
+  --tertiary-accent: #bfdbf7;
+  --background-gradient: linear-gradient(
+    170deg,
+    rgba(40, 75, 99, 1) 0%,
+    rgba(60, 110, 113, 1) 100%
+  );
+  --primary-background--tertiary-font: #fff;
+  --secondary--background--primary-font: #737373;
+  --secondary--font: #c4c4c4;
+  --navigation: #284b63;
```
Comment (marwinburesch) [inline src/app/globals.css:32]: Maybe a different name that describes more the function of that color?

File: src/app/globals.css [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
+  --primary-accent: #f25f5c;
+  --secondary-accent: #3c6e71;
+  --tertiary-accent: #bfdbf7;
+  --background-gradient: linear-gradient(
+    170deg,
+    rgba(40, 75, 99, 1) 0%,
+    rgba(60, 110, 113, 1) 100%
+  );
+  --primary-background--tertiary-font: #fff;
+  --secondary--background--primary-font: #737373;
```
Comment (marwinburesch) [inline src/app/globals.css:30]: I get the reasoning behind the double name, but I also long for a different solution :D

**Review by marwinburesch — COMMENTED** [review]
Looking good, just a few code style comments

File: src/app/globals.css (hunk 1 above)
Comment (Seborider) [reply src/app/globals.css]: resolved in [791af8f ](https://github.com/Seborider/plugs./pull/18/commits/791af8f816438c8978a17f43c04c8ed4230afd96)

File: src/app/globals.css (hunk 2 above)
Comment (Seborider) [reply src/app/globals.css]: resolved in [f3054ea](https://github.com/Seborider/plugs./pull/18/commits/f3054ead66ae8209d931bec1ca3896069475ef8b)

**Pull request merged**